### PR TITLE
List problematic file

### PR DESCRIPTION
**Pull request by ocefpaf**
Will help debug things like https://github.com/conda-forge/staged-recipes/issues/3152

### CODE REVIEW

**Comment by isuruf** [comment]
I don't know why there is a custom file copy function in `conda_smithy/feedstock_io.py`. We can just use `shutil.copy` instead of copying line by line

**Comment by ocefpaf** [comment]
Just opened https://github.com/conda-forge/conda-smithy/issues/534 thinking about that.
I will send a PR and let's see what people say.

**Comment by jakirkham** [comment]
> I don't know why there is a custom file copy function in `conda_smithy/feedstock_io.py`. We can just use `shutil.copy` instead of copying line by line

We use to, but had to change due to some CRLF characters getting added on Windows. ( https://github.com/conda-forge/conda-smithy/pull/407 )

cc @nicoddemus

File: conda_smithy/cli.py [hunk 1]
```diff
@@ -28,7 +28,13 @@ def generate_feedstock_content(target_directory, source_recipe_dir, meta):
         os.makedirs(target_recipe_dir)
     # If there is a source recipe, copy it now to the right dir
     if source_recipe_dir:
-        configure_feedstock.copytree(source_recipe_dir, target_recipe_dir)
+        try:
+            configure_feedstock.copytree(source_recipe_dir, target_recipe_dir)
+        except Exception as e:
+            import sys
+            raise type(e)(
+                str(e) + ' while copyin file %s' % source_recipe_dir
```
Comment (nicoddemus) [inline conda_smithy/cli.py]: Small typo: "copyin"

**Comment by nicoddemus** [comment]
> We use to, but had to change due to some CRLF characters getting added on Windows.

Indeed IIRC. Maybe that should be mentioned in the docstring?

**Comment by ocefpaf** [comment]
> Indeed IIRC. Maybe that should be mentioned in the docstring?

Yeah. That info should be there. I will add it in #535

**Pull request merged**